File: python/svix/models/postgres_config_patch.py
# this file is @generated
import typing as t

from .common import BaseModel


class PostgresConfigPatch(BaseModel):
    url: t.Optional[str] = None

    password: t.Optional[str] = None

    table_name: t.Optional[str] = None

    ssl_root_cert: t.Optional[str] = None

File: kotlin/lib/src/main/kotlin/models/PostgresConfigPatch.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class PostgresConfigPatch(
    val url: String? = null,
    val password: String? = null,
    val tableName: String? = null,
    val sslRootCert: String? = null,
)
